addons/l10n_ar/demo/respinsc_demo.xml

```diff
@@ -28,8 +28,12 @@
 
     <function model="res.users" name="write">
         <value eval="[ref('base.user_root'), ref('base.user_admin'), ref('base.user_demo')]"/>
-        <value eval="{'company_ids': [(4, ref('l10n_ar.company_ri'))],
-                     'company_id': ref('l10n_ar.company_ri')}"/>
+        <value eval="{'company_ids': [(4, ref('l10n_ar.company_ri'))]}"/>
+    </function>
+
+    <function model="res.users" name="write">
+        <value eval="[ref('base.user_root')]"/>
+        <value eval="{'company_id': ref('l10n_ar.company_ri')}"/>
```

Comment:
Can we avoid it for root as well?  It is a bit of an example right now that AR has a lot of demo data, but these practices are really uncommon.  Why do we need to set the company to AR inscripto after the installation?  @jjscarafia

Reply:
Yes. Writing on `base` demo data on a field that common is really not something that we want.

Reply:
Done, added this change on second commit